balloon/src/main/java/com/skydoves/balloon/custom/VectorTextViewModel.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2019 skydoves
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.skydoves.balloon.custom
+
+import android.graphics.drawable.Drawable
+import androidx.annotation.ColorRes
+import androidx.annotation.DimenRes
+import androidx.annotation.Px
+
+data class VectorTextViewModel(
```

Comment:
I think this name will make some confusion later.
If we read this code later without knowing about the `VectorTextView`, the name could be interpreted kind of `ViewModel`. How about `VectorTextViewParams`?